lib/node_modules/@stdlib/stats/base/dists/bernoulli/skewness/benchmark/benchmark.js

```diff
@@ -39,7 +39,7 @@ bench( pkg, function benchmark( b ) {
 	len = 100;
 	p = new Float64Array( len );
 	for ( i = 0; i < len; i++ ) {
-		p[ i ] = uniform( 0.0, 1.0 );
+		p[ i ] = uniform( 0.0, 10.0 );
```

Comment:
This line should not have been changed: `p` has to be a probability.

```suggestion
		p[ i ] = uniform( 0.0, 1.0 );
```